rasa/importers/default_config.yml

```diff
@@ -0,0 +1,30 @@
+# Configuration for Rasa NLU.
```

Comment:
I think the usage of `Rasa NLU` and `Rasa Core` might potentially be confusing as they refer to not longer maintained frameworks.



```suggestion
# Configuration for the Rasa NLU components.
```